### ISSUE TITLE
Email to assistant

### ISSUE THREAD

**Issue by melihaydogd** [bounswe/2021SpringGroup9#45]
Email to assistant about PS hour and customer meeting

**Comment by melihaydogd** [comment]
Email is sent

**Comment by melihaydogd** [comment]
Assistant is available on Monday. I opened a [doodle](https://doodle.com/poll/kv2tm9rw89yubw9x?utm_source=poll&utm_medium=link) to choose a suitable hour for us.

**Comment by melihaydogd** [comment]
Meeting is postponed to Wednesday. Assistant gave us a task that must be done before the meeting. We should observe how people share real life stories.

**Comment by melihaydogd** [comment]
Meeting will be held on Wednesday at 21:00.
